.github/workflows/test.yml

```diff
@@ -137,6 +137,15 @@ jobs:
         ./.github/scripts/build.sh
         ./run_reg_test.py ${{ matrix.suite }} -show_failures -j2
 
+    - name: Upload artifact
+      uses: actions/upload-artifact@v2
+      if: ${{ failure() }}
+      with:
+        name: failed_${{matrix.name}}_regression_log
+        retention-days: 3
+        path: |
+          vtr_flow/**/*.log
+          vtr_flow/**/*.out
```

Comment:
@vaughnbetz 

- If you know any files should be uploaded to the artifacts. Please feel free to comment.
- Currently, the retention of artifacts is set to 3 days. See if you need something different

Reply:
This looks great -- thanks!  I think you've captured the key ones.
To make it faster to re-run / investigate errors, we could also bring back:
*.blif (or *.pre-vpr.blif)
*.p
*.net
*.r

Will increase file sizes quite a bit, but would let us immediately examine the results of a specific compilation stage.  So if they don't cause disk problems I would bring them back (compressed).